x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -194,275 +187,272 @@ export function LayerPanel(
   ]);
 
   return (
-    <ChildDragDropProvider {...dragDropContext}>
```

Comment:
this change is only removing `ChildDragDropProvider` and removing `dragDropContext` from buttons' props.